### DISCUSSION TITLE
I have a problem and can't add-on smoothly...

### DISCUSSION THREAD

**Discussion by pearlwu0310** [nathanhoad/godot_dialogue_manager#205] [Q&A]
![image](https://github.com/nathanhoad/godot_dialogue_manager/assets/81375940/aa73d987-10f2-456e-a7ce-4f32d8215fd6)

**Comment by nathanhoad** [comment]
What version of Dialogue Manager is that? And what version of Godot?

If you open that `res://addons/dialogue_manager/constants.gd` file do you get a more specific error?

**Reply by pearlwu0310** [reply]
I'm dumb, Godot is v4.0. After updating to v4.1, it worked! Thanks
